WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -1,34 +1,78 @@
-<h1><%= @competition.name %> is confirmed</h1>
+<p>
+  Competition Announcement Team, 
+</p>
 
-<% if @competition.championships.present? %>
-  <h2 class="alert">This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %></h2>
+<p>
+  <strong><%= @confirmer.name %> has confirmed <%= link_to @competition.name, competition_url(@competition) %>.</strong>
+</p>
+
+<p class="<%= 'alert' if (@competition.start_date - Time.now.to_date).to_i <= 30 %>">
+  The competition will take place on <%= wca_date_range(@competition.start_date, @competition.end_date) %> in <%= (@competition.start_date - Time.now.to_date).to_i %> days.
+  <% if (@competition.start_date - Time.now.to_date).to_i <= 30 %>
+    There are less than 48 hours remaining for the competition to be announced. @<%= "Delegate".pluralize(@competition.delegates.count) %>: please respond to any WCAT concerns as soon as possible to ensure that the competition becomes announced. If the competition becomes less than 28 days away, the WCAT cannot announce it.
```

Comment:
How accurate is this statement with regards to timezones?

Reply:
I actually have no clue. 

How it should act is in relation to 00:00 UTC of that day, but I'm not sure how to test that.

Reply:
Added .utc to the time.now, so this now works as intended. Same with above.